Tests/Editor/Deployment/DeploymentSettingsTests.cs

```diff
@@ -1385,6 +1405,30 @@ private static void SetUpcoreApiforRefreshStackStatusSuccess(Mock<CoreApi> coreA
 
             return underTest;
         }
+        
+        private static DeploymentSettings SetUpDeleteDeployment(Mock<CoreApi> coreApiMock = null)
+        {
+            const string testGameName = "test";
+            const string testProfileName = "testProfile";
+            const string testRegion = "eu-west-1";
+
+            coreApiMock = coreApiMock ?? new Mock<CoreApi>();
+            coreApiMock.Setup(target => target.GetStackName(It.IsAny<string>()))
+                .Returns(testGameName)
+                .Verifiable();
+            coreApiMock.Setup(target => target.GetSetting(It.IsAny<SettingsKeys>()))
```

Comment:
What about verifying that the `DeleteStack` was called on core api with the profile, region, and stack name you're setting up in your mocks? That's really the only "Delete" part of the test to verify success of

Reply:
That makes sense, added.

Reply:
We should be verifying the correct data was used not verifying the method was called with any string. We're mocking the profile name and region with the state manager mock and mocking the stack name via core api, so we should verify those values are used to call DeleteStack

Reply:
Changed. Also because it turned into a proper test now, I did find out I had missed a deploymentsettings.refresh() to actually populate the profile name and region.